pkgs/by-name/by/bytecode-viewer/package.nix

```
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
+, fetchurl
+, jdk8
+, makeWrapper
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "bytecode-viewer";
+  version = "2.12";
+
+  src = fetchurl {
+    url = "https://github.com/Konloch/bytecode-viewer/releases/download/v${finalAttrs.version}/Bytecode-Viewer-${finalAttrs.version}.jar";
+    hash = "sha256-3F9maUCdfQu7pAxzWHWjmWDBd38RuxOhgZuxKReAjFw=";
+  };
+
+  dontUnpack = true;
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    runHook preInstall
+    makeWrapper ${jdk8}/bin/java $out/bin/bytecode-viewer \
```

Not sure if it needs to be pinned to JDK/JRE 8, seems to launch fine with the default one.
```suggestion
    makeWrapper ${lib.getExe jdk} $out/bin/bytecode-viewer \
```

(also requires a corresponding change at the top of the file, if you go with this)

---

Yes. We can use the default one. Fixed on top of the file. Thanks